src/plugins/data/common/search/expressions/esql.ts

```diff
@@ -341,6 +361,10 @@ export const getEsqlFn = ({ getStartDependencies }: EsqlFnArguments) => {
             type: 'datatable',
             meta: {
               type: ESQL_TABLE_TYPE,
+              query,
+              statistics: {
+                totalCount: body.values.length,
```

Comment:
@ppisljar this will return the number of rows returned from the query, is this what you want here? Where do you want to use this totalCount metric?

Reply:
yes, no specific use case, just adding this for the sake of completeness, as dsl datatable also contains this field and i think its good to align. if you think this is problematic i can omit this

Reply:
No storng opinion, is just because we have this default limit 1000 from ES that can be quite misleading, that is all